tests/ndarrays/test_ndarrays.c

```diff
@@ -184,8 +187,9 @@ int test_slicing_int(void)
     int value;
     int c_value;
 
-    x = array_init((char *)m_1, 2, m_1_shape, nd_int);
-    slice = slice_make(x, slice_data(1, 2, 1), slice_data(0, 5, 2));
+    x = array_create(2, m_1_shape, nd_int);
+    memcpy(x.raw_data, m_1, x.buffer_size);
+    slice = array_slicing(x, new_slice(1, 2, 1), new_slice(0, 5, 2));

```

Comment:
Can we use a different name instead of `slice`? What about `xview`, or simply `y`?

Reply:
will use the `xview`